ignite/handlers/checkpoint.py

```diff
@@ -253,19 +256,52 @@ def last_checkpoint(self) -> Optional[str]:
         return self._saved[-1].filename
 
     def _check_lt_n_saved(self, or_equal=False):
-        if self._n_saved is None:
+        if self.n_saved is None:
             return True
-        return len(self._saved) < self._n_saved + int(or_equal)
+        return len(self._saved) < self.n_saved + int(or_equal)
+
+    def _add_field(self, f, sep="_"):
+        return "{}{}".format(f, sep) if len(f) > 0 else f
+
+    def _get_filename_pattern(
+        self,
+        filename_prefix: str = "",
+        score_function: Optional[Callable] = None,
+        score_name: Optional[str] = None,
+        global_step_transform: Callable = None,
+        filename_pattern: Optional[str] = None,
```

Comment:
This is strange as argument, probably, there is no purpose to pass it, right ?

Reply:
You are right @vfdev-5 will update this as well.